app/renderer/components/task_card.js

```diff
@@ -66,7 +69,9 @@ module.exports = {
     },
     methods: {
         toggleRun(ev) {
-            ev.stopPropagation();
+            if (ev.target.classList.contains('btn')) {  // without this check the context menu doesn't close
```

Comment:
I don't fully understand this comment, and why this check

Reply:
Since I'm using `toggleRun` for the context menu click handler and also for the `btn` click handler it was stopping propagation for the context menu as well which kept it from closing. The context menu closes on click because the event propagates and this code runs:

https://github.com/angrykoala/gaucho/pull/197/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR37